tests/cli/test_cli.py

```diff
@@ -263,3 +264,35 @@ def test_nodes_single(self, runner):
             result = runner.invoke(cli, ["nodes", node_type])
             assert result.exit_code == 0
             assert result.output == available_nodes_msg(node_type)
+
+    def test_main_py_log_level_debug(self):
+        # setup unit test env
+        tmp_dir = os.getcwd()
+        print(f"\ntmp_dir={tmp_dir}")
+        test_config_path = Path(tmp_dir) / "test_config.yml"
```

Comment:
`Path` may not be needed if using `Path.cwd()` above.

Reply:
Removed `Path()`.